### ISSUE TITLE
correcting `use omp_lib` leads to `Error: Invalid character 0x01 at (1)`

### ISSUE THREAD

**Issue by Montagna2023** [cphyc/fortran-linter#12]
For the following code
```
Program main

  use omp_lib
  
  integer :: a1, a2, a3, a4, a5, a6, a7, a8, a9, a10000000000000000, a2222222222222222, a333333333333333333

  write (*,*) "Hello World"
End program

```

  `fortran-linter` (1.0.9) with `--syntax only` leads to  

```
  use omp_lib
  1
Warning: Should prepend with "!$" at (1).
```

  and `-i` leads to
`! $ use omp_lib` and `Error: Invalid character 0x01 at (1)`.

  It seems `use omp_lib` is ok, https://curc.readthedocs.io/en/latest/programming/OpenMP-Fortran.html, 
is this a false-positive case? Thanks.

**Comment by cphyc** [comment]
This was a bug, now fixed on master!
